actors/builtin/miner/sector_map_test.go

```diff
@@ -0,0 +1,113 @@
+package miner_test
+
+import (
+	"errors"
+	"math"
+	"testing"
+
+	"github.com/filecoin-project/go-bitfield"
+	rlepluslazy "github.com/filecoin-project/go-bitfield/rle"
+	"github.com/filecoin-project/specs-actors/actors/builtin/miner"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDeadlineSectorMap(t *testing.T) {
+	dm := make(miner.DeadlineSectorMap)
```

Comment:
Please also tests counts, iteration etc on an empty map, and map with an empty partition.